packages/hmr-plugin/src/tests/hmr-manager.spec.ts

```diff
@@ -282,4 +282,12 @@ describe('HMR Plugin', () => {
       expect(e.message).toEqual('Store not found, maybe you forgot to import the NgxsModule');
     }
   });
+
+  it('should be correct destroy old module', async () => {
+    const { appModule, webpackModule } = await hmrTestBed(AppMockModule);
+    expect(appModule._destroyed).toEqual(false);
```

Comment:
@splincode do you think it could be better if we wouldn't create `HmrModuleRef` type (just because we use it in tests) and access this property `_destroyed` via brackets like `appModule['_destroyed']`?

Reply:
Good point

Reply:
done